tinyDA/posterior.py

```diff
@@ -114,6 +114,24 @@ def update_link(self, link, bias=None):
             link.parameters, link.prior, link.model_output, likelihood, link.qoi
         )
 
+    def posterior_eval(self, parameters):
+        """
+        Parameters
+        ----------
+        parameters : numpy.ndarray
+            A numpy array of model parameters.
+
+        Returns
+        ----------
+        float
+            The posterior density evaluated at given model parameters.
```

Comment:
I'd like if this specified that it is the unnormalised posterior density, for clarity.